autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingHandler.java

```diff
@@ -45,60 +47,72 @@
     @Inject
     private ClouderaManagerTotalHostsEvaluator totalHostsEvaluatorService;
 
+    @Inject
+    private StackResponseUtils stackResponseUtils;
+
+    @Inject
+    private CloudbreakCommunicator cloudbreakCommunicator;
+
     @Override
     public void onApplicationEvent(ScalingEvent event) {
         BaseAlert alert = event.getAlert();
         Cluster cluster = clusterService.findById(alert.getCluster().getId());
         MDCBuilder.buildMdcContext(cluster);
-        scale(cluster, alert.getScalingPolicy());
+        handleClusterScaling(cluster, alert);
     }
 
-    private void scale(Cluster cluster, ScalingPolicy policy) {
-        long remainingTime = getRemainingCooldownTime(cluster);
```

Comment:
Removing this check - does it affect the evaluators other than the YarnLoadEvaluator? YarnLoadEvaluator performs this check itself. The others do not - so may end up triggering way more events?

Reply:
This actually doesnt belong here, since this just handles the event generated. The check should be done before sending the scalingevent. Currently it applies only to LoadEvaluator, since TimeBased Alert has to be applied at the scheduled time.

Reply:
Metrics and Proemetheus, if they are enabled, will end up breaking. Mind putting a note into those evaluators that they need to check the cooldown before sending events forward?
Or maybe we do that in a common base itself.

Reply:
added